test/src/kava.spec.js

```diff
@@ -519,7 +519,37 @@ describe('KavaPlugin', function() {
       player.play();
     });
 
-    it('should send VIEW event with manifest download time, segment download time and bandwidth', done => {
+    it('should send VIEW event with manifest download time, segment download time, bandwidth, networkConnectionOverhead', done => {
+      sandbox.stub(window.performance, 'getEntriesByType').callsFake(() => {
+        return [
+          {
+            name:
+              'http://il-wowza-centos-rec-01.dev.kaltura.com:8080/live/hls/p/1091/e/0_f8re4ujs/sd/10000/t/UrZp3SpmAU9V74Zf6HVFyQ/seg-61468856-s32-v1-a1.ts',
```

Comment:
use some generic name instead of this url :-)

Reply:
done